client/pages/index.tsx

```diff
@@ -7,6 +7,7 @@ import { getAllPosts } from '../lib/api'
 import Head from 'next/head'
 import { CMS_NAME } from '../lib/constants'
 import Post from '../interfaces/post'
+import Header from '../components/header'
```

Comment:
create a test to check if header is called inside of index.tsx

Reply:
ok

Reply:
Header component should be inside of layout, shouldn't it ?

Reply:
I thought would be, but next use just footer inside of layout and header inside of the posts  separatly.

Reply:
Layout use a main tag to place the components inside of layout, maybe I could try to use header inside of this main tag

Reply:
It is not defined by next,  it's defined by blog template,  and it can be changed  ;)

Reply:
i got it, i moved the header component to layout with footer